python/basix/numba_helpers.py

```diff
@@ -0,0 +1,56 @@
+try:
+    import numba
+except ImportError:
+    raise RuntimeError("You must have numba installed to use the numba helper functions.")
+
+
+@numba.njit
+def apply_dof_transformation(entity_transformations, cell_type, entity_dofs,
+                             data, block_size, cell_info):
+    if cell_type == "interval":
+        tdim = 1
+    elif cell_type == "triangle":
+        edge_count = 3
+        tdim = 2
+    elif cell_type == "quadrilateral":
+        edge_count = 4
+        tdim = 2
+    elif cell_type == "tetrahedron":
+        edge_count = 6
+        face_count = 4
+        tdim = 3
+    elif cell_type == "hexahedron":
+        edge_count = 12
+        face_count = 6
+        tdim = 3
+
+    if tdim >= 2:
+        if tdim == 3:
+            face_start = 3 * face_count
+        else:
+            face_start = 0
+
+        dofstart = 0
+        for i in entity_dofs[0]:
+            dofstart += i
+
+        for e in range(edge_count):
+            edofs = entity_dofs[1][e]
```

Comment:
What type does `entity_dofs` normally have? (Will it be fast?)

Reply:
entity_dofs[i] is an array of ints. Input type into function has been changed in the doc strings